apiserver/common/getstatus_test.go

```diff
@@ -94,9 +106,11 @@ func (s *statusGetterSuite) TestGetApplicationStatus(c *gc.C) {
 	app := s.Factory.MakeApplication(c, &factory.ApplicationParams{Status: &status.StatusInfo{
 		Status: status.Maintenance,
 	}})
-	result, err := s.getter.Status(params.Entities{[]params.Entity{{
-		app.Tag().String(),
-	}}})
+	result, err := s.getter.Status(context.Background(),
```

Comment:
nit, I feel it's a bit more readable having all args in a new line if we have at least one on the newline, i.e.:
```
	result, err := s.getter.Status(
		context.Background(),
		params.Entities{[]params.Entity{{
			unit.Tag().String(),
		}}})
```
but again, this is only my personal taste and I'm only adding this comment to think if we want to format like this when we can, not to force you reformat all this PR :)